data/location.json

```diff
@@ -171,14 +171,14 @@
     {"name":"Fort Redwater","x":177026,"y":-61028,"formId":"0x000174c0","icon":"Fort"},
     {"name":"Fort Cuptor","x":180701,"y":-27563,"formId":"0x000174d1","icon":"Fort"},
     {"name":"Oblivion Gates", "elements":[
-		{"name":"Oblivion Gate - Anvil","x":-202117,"y":-23049,"formId":"0x000cd414","icon":"Gate"},
-		{"name":"Oblivion Gate - Bruma","x":30043,"y":143344,"formId":"0x0000cefa","icon":"Gate"},
-		{"name":"Oblivion Gate - Kvatch","x":-144798,"y":-17910,"formId":"0x0000cef7","icon":"Gate"},
-		{"name":"Oblivion Gate - Chorrol","x":-63444,"y":84905,"formId":"0x0000cefb","icon":"Gate"},
-		{"name":"Oblivion Gate - Skingrad","x":-52058,"y":2307,"formId":"0x0000cefd","icon":"Gate"},
-		{"name":"Oblivion Gate - Bravil","x":71302,"y":-27351,"formId":"0x0000cef9","icon":"Gate"},
-		{"name":"Oblivion Gate - Leyawiin","x":103901,"y":-138643,"formId":"0x0000cefc","icon":"Gate"},
-		{"name":"Oblivion Gate - Cheydinhal","x":109041,"y":96580,"formId":"0x0000cef8","icon":"Gate"},
+		{"name":"Oblivion Gate - Anvil - Fixed, 2 Fame, No Reroll","x":-202117,"y":-23049,"formId":"0x000cd414","icon":"Gate"},
+		{"name":"Oblivion Gate - Bruma - Fixed","x":30043,"y":143344,"formId":"0x0000cefa","icon":"Gate"},
+		{"name":"Oblivion Gate - Kvatch - Fixed","x":-144798,"y":-17910,"formId":"0x0000cef7","icon":"Gate"},
+		{"name":"Oblivion Gate - Chorrol - Fixed, 2 Fame, No Reroll","x":-63444,"y":84905,"formId":"0x0000cefb","icon":"Gate"},
+		{"name":"Oblivion Gate - Skingrad - Fixed, 2 Fame, No Reroll","x":-52058,"y":2307,"formId":"0x0000cefd","icon":"Gate"},
+		{"name":"Oblivion Gate - Bravil - Fixed, 2 Fame, No Reroll","x":71302,"y":-27351,"formId":"0x0000cef9","icon":"Gate"},
+		{"name":"Oblivion Gate - Leyawiin - Fixed, 2 Fame, No Reroll","x":103901,"y":-138643,"formId":"0x0000cefc","icon":"Gate"},
+		{"name":"Oblivion Gate - Cheydinhal - Fixed, No Reroll","x":109041,"y":96580,"formId":"0x0000cef8","icon":"Gate"},
 		{"name":"Oblivion Gate - Fixed","x":-187164,"y":14116,"formId":"0x0000cd09","icon":"Gate"},
```

Comment:
Try putting random gates in their own section and see if that breaks the checklist.

Reply:
It shows up like this, 

![Random Gate label](https://user-images.githubusercontent.com/47337746/148442960-3550afb6-198c-49f5-9e3b-a8b804ec1da7.PNG)

thanks for the reminder to check the checklist. Forgot about how the things we do in json affects it.